### PR TITLE
Add tables, badges, alerts & form

### PR DESCRIPTION
**Pull request by atav1k**

### CODE REVIEW

**Comment by atav1k** [comment]
Checked the vanilla gulp build & all good.

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 Fortune helps prototype sites by providing a set of non-blocking utilities and basic components.
 
-- ⚡️  Light-weight at 10kb
+- ⚡️  Light-weight at < 10kb gzipped
```
Comment (chasestarr) [inline README.md]: 👌

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "fortune-css",
-  "version": "0.0.2",
+  "version": "0.0.3",
```
Comment (chasestarr) [inline package.json]: 🌥

Comment (chasestarr) [reply package.json]: did you publish?

File: src/form.css [hunk 3]
```diff
@@ -0,0 +1,492 @@
...
+  width: 100%;
+  height: var(--input-height);
+  line-height: var(--input-height);
+  vertical-align: middle;
+
+  &:focus {
+    outline: none;
+  }
+
+  /* Webkit */
```
Comment (chasestarr) [inline src/form.css]: should this be here if end-users could use the prefixed css?

**Comment by chasestarr** [comment]
this lgtm

File: package.json (hunk 2 above)
Comment (atav1k) [reply package.json]: No, haven't done npm publishing yet

File: src/form.css (hunk 3 above)
Comment (atav1k) [reply src/form.css]: Looks like its only a webkit implementation for range slider

**Comment by atav1k** [comment]
Note that both gulp builds remove -webkit & -moz specs

File: src/tab.css [hunk 4]
```diff
@@ -0,0 +1,24 @@
+.tabs {
+  display: flex;
+}
+
+.tabs > a {
```
Comment (atav1k) [inline src/tab.css]: Blocks utility styling in #3 because of higher specificity

**Pull request merged**